File: bms/go/go-edlib/OrderedMap_Len/OrderedMap_Len.go
package test

var ()

const ()

func (p OrderedMap) Len() int { return len(p) }

type pair struct {
	Key   string
	Value float32
}

// OrderedMap is a slice of pairs type with string keys and float values.
// It implement sorting methods by values.
type OrderedMap []pair

File: translations_only/success/go/go-edlib/OrderedMap_Len/claude3/Restart/OrderedMap_Len.rs

// OrderedMap is a Vec of pair type with String keys and f32 values.
// It implements sorting methods by values.
struct OrderedMap(Vec<pair>);

struct pair {
    Key: String,
    Value: f32,
}

fn len(p: &OrderedMap) -> isize {
    p.0.len() as isize
}
